### PR TITLE
Update cs.js - Added some contextual keywords

### PR DESCRIPTION
**Pull request by martinhodler**
# References
- https://msdn.microsoft.com/en-us/library/x53a06bb.aspx

# Notes
The only keyword which is not on this page is the "nameof"-Keyword.
But I never saw the keyword "when"

### CODE REVIEW

File: src/languages/cs.js [hunk 1]
```diff
@@ -8,16 +8,16 @@ function(hljs) {
   var KEYWORDS = {
     keyword:
       // Normal keywords.
-      'abstract as base bool break byte case catch char checked const continue decimal dynamic ' +
```
Comment (martinhodler) [inline src/languages/cs.js]: Moved dynamic keyword to the contextual keywords section

File: src/languages/cs.js [hunk 2]
```diff
@@ -8,16 +8,16 @@ function(hljs) {
...
+      'abstract as base bool break byte case catch char checked const continue decimal ' +
       'default delegate do double else enum event explicit extern finally fixed float ' +
-      'for foreach goto if implicit in int interface internal is lock long when ' +
```
Comment (martinhodler) [inline src/languages/cs.js]: Removed when keyword

File: src/languages/cs.js [hunk 3]
```diff
@@ -8,16 +8,16 @@ function(hljs) {
...
       'object operator out override params private protected public readonly ref sbyte ' +
       'sealed short sizeof stackalloc static string struct switch this try typeof ' +
-      'uint ulong unchecked unsafe ushort using virtual volatile void while async ' +
+      'uint ulong unchecked unsafe ushort using virtual void volatile while ' +
       'nameof ' +
       // Contextual keywords.
-      'ascending descending from get group into join let orderby partial select set value var ' +
-      'where yield',
+      'add alias ascending async await by descending dynamic equals from get global group into join ' +
+      'let on orderby partial remove select set value var where yield',
```
Comment (martinhodler) [inline src/languages/cs.js]: Added some contextual keywords like by, global, on, etc.

**Comment by isagalaev** [comment]
Thanks for updating the language!

**Pull request merged**